deploy/cd-namespace/chart/values.yaml

```diff
@@ -6,41 +6,66 @@
 serviceAccountName: 'pipeline'
 
 # Bitbucket
+# Bitbucket URL (including scheme). Example: https://bitbucket.example.com.
 bitbucketUrl: ''
+# Bitbucket username. Example: cd_user.
 bitbucketUsername: ''
+# Bitbucket personal access token (PAT) in clear text. This token needs to be
+# created in Bitbucket for the user identified by "bitbucketUsername".
 bitbucketAccessToken: ''
+# Shared secret between webhooks in Bitbucket and the event listener. Create
+# a random secret and enter it here, then use it later on as the secret in any
+# webhook you setup.
 bitbucketWebhookSecret: ''
```

Comment:
Should this be a plaintext or base64-encoded value?

Reply:
275937e